package.json

```diff
@@ -34,34 +34,37 @@
     ]
   },
   "dependencies": {
-    "styled-components": "^5.2.3"
+    "styled-components": "^6.1.8"
   },
   "peerDependencies": {
-    "react": ">=16 <=17",
-    "react-dom": ">=16 <=17"
+    "react": ">=16 <=18",
+    "react-dom": ">=16 <=18"
```

Comment:
You could consider making it simply `>=16` and then we'd only have to change it if it breaks in a future version of React. 🤷

Reply:
I kind of like the idea that we have to approve new versions of React. We can't really revise already released library versions, so if we say it supports everything 16 or higher we'd have a "lying" module sitting around.